x/zktx/module/cli.go

```diff
@@ -69,30 +70,30 @@ func (am AppModule) GetTxCmd() *cobra.Command {
 				return fmt.Errorf("invalid state changes")
 			}
 
-			stateChanges := make([]*zktx.StateChange, 0, len(args)/2)
+			payloads := make([]*zktx.Payload, 0, len(args)/2)
 			for i := 0; i < len(args); i += 2 {
-				proof, err := GetBinaryValue(args[i+1])
+				data, err := GetBinaryValue(args[i+1])
 				if err != nil {
 					return err
 				}
-				stateChanges = append(stateChanges, &zktx.StateChange{
+				payloads = append(payloads, &zktx.Payload{
 					ContractName: args[i],
-					Proof:        proof,
+					Data:         data,
 				})
 			}
 
-			msg := &zktx.MsgExecuteStateChanges{
-				StateChanges: stateChanges,
+			msg := &zktx.MsgPublishPayloads{
+				Payloads: payloads,
 			}
 
 			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
 		},
 	})
 
 	txCmd.AddCommand(&cobra.Command{
-		Use:   "verify [contract_name] [proof]",
-		Short: "Verify a proof",
-		Args:  cobra.MinimumNArgs(2),
+		Use:   "verify [tx_hash] [index] [contract_name] [payload_hash] [proof]",
```

Comment:
Maybe use something like 
```suggestion
		Use:   "prove [tx_hash] [index] [contract_name] [payload_hash] [proof]",
```

Reply:
Done